include/ck/utility/f8_utils.hpp

```diff
@@ -141,81 +126,70 @@ __host__ __device__ Y run_cast_to_f8(X x, uint32_t rng)
 
     // check if x is 0.0 or -0.0
     if(exponent == 0 && mantissa == 0)
-        return negative_zero_nan ? 0 : (sign << (f8_exp + f8_mant));
-    mantissa &= (1 << f8_mant) - 1;
-    return (sign << (f8_exp + f8_mant)) | (exponent << f8_mant) | mantissa;
+        return negative_zero_nan ? 0 : (sign << (out_exp + out_mant));
+    mantissa &= (1 << out_mant) - 1;
+    return (sign << (out_exp + out_mant)) | (exponent << out_mant) | mantissa;
 }
 
 template <typename X, typename Y, bool negative_zero_nan>
 __host__ __device__ Y run_cast_from_f8(X x)
 {
     // check data type
-    constexpr bool is_half  = std::is_same<Y, half_t>::value;
-    constexpr bool is_float = std::is_same<Y, float>::value;
-    constexpr bool is_f8_t  = std::is_same<X, f8_t>::value;
-    constexpr bool is_bf8_t = std::is_same<X, bf8_t>::value;
+    // constexpr bool is_half  = std::is_same<Y, half_t>::value;
+    // constexpr bool is_float = std::is_same<Y, float>::value;
+    // constexpr bool is_f8_t  = std::is_same<X, f8_t>::value;
+    // constexpr bool is_bf8_t = std::is_same<X, bf8_t>::value;
```

Comment:
Leftovers.

Reply:
Oops, thanks for noticing, removed in [6de7d10](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/792/commits/6de7d10dc96a679dd9b3d59d66a1201c9930092e)